piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,6 +77,20 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def verify_operator_installed(self, operator_name, namespace):
+        """
+        Check if operator is installed and returned true or false
+        :param operator_name: name of the operator.
+        :param channel_name: name of the channel
+        return: object of CSV
+        """  
+        csv = ClusterServiceVersion(self.kube_config_file)
```

Comment:
Maybe we should move csv with the other global object declarations in the constructor for consistency.

Reply:
Done.